packages/block-editor/src/components/block-preview/index.js

```diff
@@ -55,9 +55,16 @@ function ScaledBlockPreview( { blocks, viewportWidth } ) {
 
 				// Hack: we need  to reset the scaled elements margins
 				previewElement.style.marginTop = '0';
+
+				onReady( {
+					scale,
+					position: { x: offsetX * scale, y: offsetY },
+				} );
 			} else {
 				const containerElementRect = containerElement.getBoundingClientRect();
-				setPreviewScale( containerElementRect.width / viewportWidth );
+				const scale = containerElementRect.width / viewportWidth;
+				setPreviewScale( scale );
+				onReady( { scale, ref: previewRef } );
```

Comment:
Now we don't need to pass the ref in the event anymore right?

Reply:
Added a comment here (edited as well, sorry)
https://github.com/WordPress/gutenberg/pull/17242#discussion_r327164265